src/components/calculator.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
```

Comment:
- [ ] For this module, we need to ensure that the calculator component has at least one child. We can do so by refactoring the buttons into its own component, and using it inside the calculator component.